pkg/mattermost/mattermost.go

```diff
@@ -0,0 +1,236 @@
+package mattermost
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/infracloudio/botkube/pkg/config"
+	"github.com/infracloudio/botkube/pkg/execute"
+	"github.com/infracloudio/botkube/pkg/logging"
+	"github.com/mattermost/mattermost-server/model"
+)
+
+var client *model.Client4
+var webSocketClient *model.WebSocketClient
+var botUser *model.User
+var botTeam *model.Team
+var botChannel *model.Channel
+
+const (
+	// BotName stores Botkube details
+	BotName = "botkube"
+	// BotPassword stores password for Botkube
+	BotPassword = "BotPassword@123"
```

Comment:
hardcoding ?

Reply:
We will mention in the Readme that this is the default password for Botkube user. It is recommended to change the password in settings.

Reply:
User creation will be done by the admin itself. So no need of hardcoding password.